site/_docs/history.md

```diff
@@ -31,6 +31,11 @@ Downloads are available on the
 ## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.23.0">1.23.0</a> / under development
 {: #v1-23-0}
 
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using Oracle JDK 8, 9, 10, 11, 12, 13, 14 and OpenJDK 8, 9, 10, 11, 12, 13, 14;
+Guava versions 19.0 to 28.2-jre; Apache Flink 1.10.0;
```

Comment:
> 19.0 to 28.2-jre;

Are you sure all the versions in-between are really tested? Where's the CI?

Reply:
Not sure where 'Apache Flink' came from. There used to be 'Apache Druid', because we have a Druid adapter, and we tested against a particular version of Druid. We don't depend on any Druid maven artifacts - because we speak to Druid using REST - so the version is not apparent by simply looking at our maven dependencies.

Flink doesn't belong on this list because we don't test against Flink. Flink tests against us.

@danny0405 Can you comment?

Reply:
I test Flink manually when upgrade Calcite for Flink and prepare for Calcite release. And i put it in the release note.

Reply:
Calcite does not rely on Flink, so I guess we should remove Flink from the list here because it might confuse the reader.

JDK releases do not put words like "this JDK release was tested with Calcite 1.22"